### PR TITLE
[FEATURE] change editor spacing style

### PR DESCRIPTION
**Pull request by HLTVProxy**
- change editor spacing style
- add two icons for spacing style
- add translated text

### CODE REVIEW

**Comment by HLTVProxy** [comment]
[【頁面模組】內外距數值調整方式優化](https://trello.com/c/vBTLVZft/3369-%E3%80%90%E9%A0%81%E9%9D%A2%E6%A8%A1%E7%B5%84%E3%80%91%E5%85%A7%E5%A4%96%E8%B7%9D%E6%95%B8%E5%80%BC%E8%AA%BF%E6%95%B4%E6%96%B9%E5%BC%8F%E5%84%AA%E5%8C%96)

File: src/components/craft/translation.ts [hunk 1]
```diff
@@ -158,6 +152,15 @@ const craftMessages = {
...
   }),
+  SpaceStyleInput: defineMessages({
+    spacing: { id: 'craft.inputs.spacing', defaultMessage: 'spacing' },
+    margin: { id: 'craft.inputs.margin', defaultMessage: 'margin' },
+    padding: { id: 'craft.inputs.padding', defaultMessage: 'padding' },
+    top: { id: 'craft.label.top', defaultMessage: 'top' },
+    right: { id: 'craft.label.right', defaultMessage: 'right' },
+    bottom: { id: 'craft.label.bottom', defaultMessage: 'bottom' },
+    left: { id: 'craft.label.left', defaultMessage: 'left' },
+  }),
```
Comment (sunnylochun) [inline src/components/craft/translation.ts:163]: id should be craft.SpaceStyleInput.xxx

**Review by sunnylochun — APPROVED** [review]

File: src/components/craft/translation.ts (hunk 1 above)
Comment (HLTVProxy) [reply src/components/craft/translation.ts]: @sunnylochun I had fixed. You can see commit below this comment

File: src/components/craft/inputs/SpaceStyleInput.tsx [hunk 2]
```diff
@@ -1,45 +1,152 @@
...
+      let padding = paddingArr.join(' ')
+      onChange?.({ ...value, padding })
+    }
+  }
+
+  const handleKeyUp = (e: React.KeyboardEvent<HTMLInputElement>, order: number) => {
+    onSpacingChange(e.currentTarget.value, order)
+  }
+
   const { formatMessage } = useIntl()
```
Comment (Ponchimeow) [inline src/components/craft/inputs/SpaceStyleInput.tsx:70]: NITS:  can move up to line 41

**Review by Ponchimeow — APPROVED** [review]

File: src/components/craft/inputs/SpaceStyleInput.tsx (hunk 2 above)
Comment (HLTVProxy) [reply src/components/craft/inputs/SpaceStyleInput.tsx:70]: @Ponchimeow  I had moved. You can see commit below this comment

**Pull request merged**